config.py

```diff
@@ -21,12 +21,16 @@
 SECURITY_POST_REGISTER_VIEW = "/admin/"
 
 # Flask-Security features
-SECURITY_CONFIRMABLE = True
```

Comment:
Can we change this to be true if an environment variable is present (eg 'ALLOW_USER_REGISTRATION')

Reply:
Enabling SECURITY_REGISTERABLE must enable SECURITY_CONFIRMABLE as well.

Reply:
added